pytype/tools/analyze_project/main.py

```diff
@@ -75,12 +75,15 @@ def main():
     import_graph = importlab.graph.ImportGraph.create(
         env, conf.inputs, trim=True)
     unreadable_inputs = conf.inputs & import_graph.unreadable_files
-    assert not unreadable_inputs, '\n  '.join(
+    unreadable_python_inputs = {file for file in unreadable_inputs if file.endswith('.py')}
+    assert not unreadable_python_inputs, '\n  '.join(
         ['Unparseable in Python %s:' % conf.python_version] +
-        sorted(unreadable_inputs))
+        sorted(unreadable_python_inputs))
   except Exception as e:  # pylint: disable=broad-except
     logging.critical('Cannot parse input files:\n%s', str(e))
```

Comment:
The "str(e)" isn't needed ("%s" in the format takes care of that).
[Yes, I know it's not part of your changes, but improving code hygiene  when you see it is usually a good thing.]

I would also suggest removing the "\n" -- it puts the error message on a separate line in the log, but without context, so more difficult to use automation to scan the logs.

Reply:
Sure, Thanks for the nudge

Reply:
Done!!